<?xml version="1.0" encoding="UTF-8"?><testrun name="ActivityClientTest" project="exercise-services" tests="2" started="2" failures="0" errors="2" ignored="0">
  <testsuite name="com.pluralsight.client.ActivityClientTest" time="2.946">
    <testcase name="testGet" classname="com.pluralsight.client.ActivityClientTest" time="1.884">
      <error>javax.ws.rs.ProcessingException: java.net.ConnectException: Connection refused: connect&#13;
	at org.glassfish.jersey.client.HttpUrlConnector.apply(HttpUrlConnector.java:205)&#13;
	at org.glassfish.jersey.client.ClientRuntime.invoke(ClientRuntime.java:215)&#13;
	at org.glassfish.jersey.client.JerseyInvocation$2.call(JerseyInvocation.java:671)&#13;
	at org.glassfish.jersey.internal.Errors.process(Errors.java:315)&#13;
	at org.glassfish.jersey.internal.Errors.process(Errors.java:297)&#13;
	at org.glassfish.jersey.internal.Errors.process(Errors.java:228)&#13;
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:422)&#13;
	at org.glassfish.jersey.client.JerseyInvocation.invoke(JerseyInvocation.java:667)&#13;
	at org.glassfish.jersey.client.JerseyInvocation$Builder.method(JerseyInvocation.java:396)&#13;
	at org.glassfish.jersey.client.JerseyInvocation$Builder.get(JerseyInvocation.java:296)&#13;
	at com.pluralsight.client.ActivityClient.get(ActivityClient.java:27)&#13;
	at com.pluralsight.client.ActivityClientTest.testGet(ActivityClientTest.java:18)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: java.net.ConnectException: Connection refused: connect&#13;
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)&#13;
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)&#13;
	at java.net.PlainSocketImpl.connect(Unknown Source)&#13;
	at java.net.SocksSocketImpl.connect(Unknown Source)&#13;
	at java.net.Socket.connect(Unknown Source)&#13;
	at sun.net.NetworkClient.doConnect(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.openServer(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.openServer(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.&lt;init&gt;(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.New(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.New(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.connect(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)&#13;
	at java.net.HttpURLConnection.getResponseCode(Unknown Source)&#13;
	at org.glassfish.jersey.client.HttpUrlConnector._apply(HttpUrlConnector.java:296)&#13;
	at org.glassfish.jersey.client.HttpUrlConnector.apply(HttpUrlConnector.java:203)&#13;
	... 34 more&#13;
&#13;
</error>
    </testcase>
    <testcase name="testGetAll" classname="com.pluralsight.client.ActivityClientTest" time="1.061">
      <error>javax.ws.rs.ProcessingException: java.net.ConnectException: Connection refused: connect&#13;
	at org.glassfish.jersey.client.HttpUrlConnector.apply(HttpUrlConnector.java:205)&#13;
	at org.glassfish.jersey.client.ClientRuntime.invoke(ClientRuntime.java:215)&#13;
	at org.glassfish.jersey.client.JerseyInvocation$3.call(JerseyInvocation.java:693)&#13;
	at org.glassfish.jersey.internal.Errors.process(Errors.java:315)&#13;
	at org.glassfish.jersey.internal.Errors.process(Errors.java:297)&#13;
	at org.glassfish.jersey.internal.Errors.process(Errors.java:228)&#13;
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:422)&#13;
	at org.glassfish.jersey.client.JerseyInvocation.invoke(JerseyInvocation.java:689)&#13;
	at org.glassfish.jersey.client.JerseyInvocation$Builder.method(JerseyInvocation.java:405)&#13;
	at org.glassfish.jersey.client.JerseyInvocation$Builder.get(JerseyInvocation.java:301)&#13;
	at com.pluralsight.client.ActivityClient.getAll(ActivityClient.java:34)&#13;
	at com.pluralsight.client.ActivityClientTest.testGetAll(ActivityClientTest.java:28)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: java.net.ConnectException: Connection refused: connect&#13;
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)&#13;
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)&#13;
	at java.net.PlainSocketImpl.connect(Unknown Source)&#13;
	at java.net.SocksSocketImpl.connect(Unknown Source)&#13;
	at java.net.Socket.connect(Unknown Source)&#13;
	at sun.net.NetworkClient.doConnect(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.openServer(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.openServer(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.&lt;init&gt;(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.New(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.New(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.connect(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)&#13;
	at java.net.HttpURLConnection.getResponseCode(Unknown Source)&#13;
	at org.glassfish.jersey.client.HttpUrlConnector._apply(HttpUrlConnector.java:296)&#13;
	at org.glassfish.jersey.client.HttpUrlConnector.apply(HttpUrlConnector.java:203)&#13;
	... 34 more&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
